cpp/velox/memory/VeloxColumnarBatch.cc

```diff
@@ -47,20 +48,13 @@ void VeloxColumnarBatch::ensureFlattened() {
   if (flattened_ != nullptr) {
     return;
   }
-  auto startTime = std::chrono::steady_clock::now();
-  // Make sure to load lazy vector if not loaded already.
+
+  ScopedTimer timer(&exportNanos_);
```

Comment:
curious could we use velox's `MicrosecondTimer` to avoid repeated implementation of same functionality.